pocs/tests/test_observatory.py

```diff
@@ -156,6 +156,19 @@ def test_add_remove_scheduler(config, observatory, caplog):
         observatory.add_scheduler("scheduler")
 
 
+def test_set_dome(config_with_simulated_dome):
+    conf = config_with_simulated_dome.copy()
+    dome = create_dome_from_config(conf)
+    obs = Observatory(config=conf, dome=dome)
+    assert obs.has_dome is True
+    obs.set_dome()
```

Comment:
The more I see this the more I dislike it in term of removing the dome.

I'm thinking we should make this a positional parameter rather than a keyword, so that the user has to specifically say `obs.set_dome(None)` to remove the dome.  Then `obs.set_dome()` with nothing would be an error.  Okay for now but I will make an Issue out of this.

Reply:
#865 created.